src/main/java/dev/jbang/cli/Alias.java

```diff
@@ -28,6 +30,8 @@
 	protected Path getCatalog() {
 		if (global) {
 			return Settings.getAliasesFile();
+		} else if (catalogFile != null && Files.isDirectory(catalogFile)) {
```

Comment:
Almost, we still need to check for the `catalogFile/.jbang/jbang-catalog.json` file as well. Also remember that in case that the file doesn't exist we still need to return `catalogFile/jbang-catalog.json` so `alias add` knows what file to.

Reply:
Oops 🤦

Reply:
@quintesse Fixed.